sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/CrashesTest.java

```diff
@@ -1513,8 +1473,10 @@ public void minidumpFilePathNull() throws Exception {
         mockStatic(ErrorLogHelper.class);
         mockStatic(ErrorAttachmentLog.class);
         ErrorReport errorReport = new ErrorReport();
-        errorReport.setThrowable(new NativeException());
-        when(ErrorLogHelper.getErrorReportFromErrorLog(any(ManagedErrorLog.class), any(Throwable.class))).thenReturn(errorReport);
+        Device device = new Device();
+        device.setWrapperSdkName(WRAPPER_SDK_NAME_NDK);
```

Comment:
I would use mock here since it's _unit_ test. Not a big deal for model objects though